src/constraints/field.rs

```diff
@@ -279,81 +267,10 @@ pub fn if_else_inner<B: Backend>(
         ConstOrCell::Cell(cond) => *cond,
```

Comment:
why return a value here?

Reply:
ah, it shouldn't.
fix: a9e936e8546fcc2c445947b52ddca4af26d6ddd2